src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -291,6 +292,9 @@ public boolean isACar(Entity cart) {
 		if((cart.getType().toString().toUpperCase().contains("MINECART") && cart.getType().toString().length() > 8) || cart.getType().toString().toUpperCase().contains("BOAT")) {
 			return false; //Not a car but a Minecart_Something (only useful when derailed)
 		}
+		if(cart instanceof Animals) {
+			return false;	//Animals are not cars...
```

Comment:
Why is this needed?

Reply:
Animal Vehicles like horses, pigs or striders were handled as cars when they were on roadblocks/the entire time when roadblocks were turned off

Reply:
Ah okay makes sense